drivers/pulseaudio/audio_driver_pulseaudio.cpp

```diff
@@ -726,6 +726,7 @@ Error AudioDriverPulseAudio::init_input_device() {
 	int input_buffer_size = input_buffer_frames * spec.channels;
 
 	pa_buffer_attr attr;
```

Comment:
I'd suggest also zeroing the struct so at least we have sane defaults if we end up relying on some of the other struct fields.
```suggestion
	pa_buffer_attr attr = {};
```